Classes/Private/Manager/Audio/OCTAudioEngine.m

```diff
@@ -37,31 +40,41 @@ - (instancetype)init
     _ioUnitDescription.componentFlags = 0;
     _ioUnitDescription.componentFlagsMask = 0;
 
+    TPCircularBufferInit(&_buffer, kBufferLength);
+
     NewAUGraph(&_processingGraph);
 
-    AUGraphAddNode(_processingGraph,
+    AUGraphAddNode(self.processingGraph,
                    &_ioUnitDescription,
                    &_ioNode);
 
-    AUGraphOpen(_processingGraph);
+    AUGraphOpen(self.processingGraph);
 
-    AUGraphNodeInfo(_processingGraph, _ioNode, NULL, &_ioUnit);
+    AUGraphNodeInfo(self.processingGraph, self.ioNode, NULL, &_ioUnit);
```

Comment:
But for `init` method it is better [to use ivar](https://stackoverflow.com/questions/5932677/initializing-a-property-dot-notation/5932733#5932733) :-)